buildSrc/build.gradle.kts

```diff
@@ -1,11 +1,24 @@
+plugins {
+    `kotlin-dsl`
+}
+
+group = "com.simtop.billionbeers.buildlogic"
+version = "1.0-SNAPSHOT"
+
 repositories {
+    google()
     mavenCentral()
+    gradlePluginPortal()
 }
 
-plugins {
-    `kotlin-dsl`
+dependencies {
+    implementation("com.android.tools.build:gradle:8.13.1")
+    implementation("com.android.tools.build:gradle-api:8.13.1")
+    //implementation("org.jetbrains.kotlin.jvm:org.jetbrains.kotlin.jvm.gradle.plugin:2.1.21")
```

Comment:
Check if it's needed or we can delete, also check if we can use libs directly here.